pkg/handler3/handler.go

```diff
@@ -222,27 +219,13 @@ func (o *OpenAPIV3Group) UpdateSpec(specBytes []byte) (err error) {
 	o.rwMutex.Lock()
 	defer o.rwMutex.Unlock()
 
-	specPb, err := ToV3ProtoBinary(specBytes)
-	if err != nil {
-		return err
-	}
-
-	specPbGz := toGzip(specPb)
```

Comment:
would you mind deleting the toGzip function as well?

Reply:
Whoops, didn't see that it was unused now - fixed, thanks!